packages/grid/_modules_/grid/models/gridOptions.tsx

```diff
@@ -389,64 +392,65 @@ export interface GridOptions {
    * @param event [[MuiEvent<React.SyntheticEvent>]].
    */
-  onRowDoubleClick?: (param: GridRowParams, event: MuiEvent<React.SyntheticEvent>) => void;
+  onRowDoubleClick?: (params: GridRowParams, event: MuiEvent<React.SyntheticEvent>) => void;
   /**
    * Callback fired when a mouseover event comes from a row container element.
-   * @param param With all properties from [[GridRowParams]].
+   * @param params With all properties from [[GridRowParams]].
    * @param event [[MuiEvent<React.SyntheticEvent>]].
    */
-  onRowOver?: (param: GridRowParams, event: MuiEvent<React.SyntheticEvent>) => void;
+  onRowOver?: (params: GridRowParams, event: MuiEvent<React.SyntheticEvent>) => void;
   /**
    * Callback fired when a mouseout event comes from a row container element.
-   * @param param With all properties from [[GridRowParams]].
+   * @param params With all properties from [[GridRowParams]].
    * @param event [[MuiEvent<React.SyntheticEvent>]].
    */
-  onRowOut?: (param: GridRowParams, event: MuiEvent<React.SyntheticEvent>) => void;
+  onRowOut?: (params: GridRowParams, event: MuiEvent<React.SyntheticEvent>) => void;
   /**
    * Callback fired when a mouse enter event comes from a row container element.
-   * @param param With all properties from [[GridRowParams]].
+   * @param params With all properties from [[GridRowParams]].
    * @param event [[MuiEvent<React.SyntheticEvent>]].
    */
-  onRowEnter?: (param: GridRowParams, event: MuiEvent<React.SyntheticEvent>) => void;
+  onRowEnter?: (params: GridRowParams, event: MuiEvent<React.SyntheticEvent>) => void;
   /**
    * Callback fired when a mouse leave event comes from a row container element.
-   * @param param With all properties from [[GridRowParams]].
+   * @param params With all properties from [[GridRowParams]].
    * @param event [[MuiEvent<React.SyntheticEvent>]].
    */
-  onRowLeave?: (param: GridRowParams, event: MuiEvent<React.SyntheticEvent>) => void;
+  onRowLeave?: (params: GridRowParams, event: MuiEvent<React.SyntheticEvent>) => void;
   /**
    * Callback fired when the grid is resized.
-   * @param param With all properties from [[GridResizeParams]].
+   * @param params With all properties from [[GridResizeParams]].
    * @param event [[MuiEvent<{}>]].
    */
-  onResize?: (param: GridResizeParams, event: MuiEvent<{}>) => void;
+  onResize?: (params: GridResizeParams, event: MuiEvent<{}>) => void;
   /**
    * Callback fired when the selection state of one or multiple rows changes.
    * @param selectionModel With all the row ids [[GridRowId]][].
    */
   onSelectionModelChange?: (selectionModel: GridRowId[]) => void;
   /**
    * Callback fired when the sort model changes before a column is sorted.
-   * @param param With all properties from [[GridSortModel]].
+   * @param model With all properties from [[GridSortModel]].
    */
   onSortModelChange?: (model: GridSortModel) => void;
   /**
    * Callback fired when the state of the grid is updated.
+   * @private
```

Comment:
At this moment, we don't have automatically generated docs for components but this tag is [discouraged](https://typedoc.org/guides/doccomments/#%40public%2C-%40protected%2C-and-%40private
) by TypeDoc.

```suggestion
   * @internal
```